### ISSUE TITLE
unbreak steamtrades.com

### ISSUE THREAD

**Issue by Mx772** [safing/intel-data#45]
Looks like `steamtrades.com` is on one of the lists.

Wireshark for ref:
```
.............steamtrades.com..................steamtrades.com......steamtrades.com................info
portmaster..........21blocked: steamtrades.com. in activated lists OISD.info
portmaster..........10steamtrades.com. is blocked by filter lists OISD
```

But looking at OISD and reporting it here:
https://oisd.nl/report

It notes that:
![image](https://user-images.githubusercontent.com/9059161/220700273-9a259177-0854-4398-a2a9-c8981208b446.png)

**Comment by davegson** [comment]
this is an odd one as I could not reproduce it. PM did not block it for me - but ofc I did not test it immediately as you posted it. I mean it clearly _was_ blocked by the OISD list within PM as shown in the logs. Maybe the list was quickly updated by the maintainer? Is this still an issue for you?

Unsure how likely this would be to become a false block in the future - do you think it's likely enough to add it to the list?

**Comment by Mx772** [comment]
I wonder if OISD unblocked it soon after I found it? 
Or possibly their 'report site' is just a pipeline that automatically unblocks it or something?

I doubt it'd become a false block in the future as it really had no reason to be in the list in the first place. 

Unfortunately I had to uninstall PM as I was getting a good 3 second delay for every website to load (assuming DNS resolution? Not really sure). As soon as I uninstalled the resolution timing was much better which is a shame because I did like it. 


TL;DR: Probably good to close, doubt it'll show up again.

**Comment by davegson** [comment]
> Unfortunately I had to uninstall PM as I was getting a good 3 second delay for every website to load (assuming DNS resolution? Not really sure). As soon as I uninstalled the resolution timing was much better which is a shame because I did like it.

thanks for the feedback - sometimes related software slows things down. Not saying we cannot improve - we're trying to do so step by step. Thanks for giving it a go!
